test.js

```diff
@@ -93,7 +93,7 @@ describe('cpFile()', function () {
 
 	it('should not create dest on unreadable src', function (cb) {
 		cpFile('node_modules', 'tmp', function (err) {
-			assert(err.code === 'EISDIR');
+			assert(err.cause().code === 'EISDIR');
```

Comment:
To be honest, I have not started a thorough research. I wanted to find out more what your general opinion on such things are.